lib/keychain.c

```diff
@@ -362,7 +362,7 @@ uint32_t keychain_get_block_size(enum keychain_hash_algo key)
 	return algo_info[key].block;
 }
 
-uint32_t keychain_get_hash_len(enum keychain_hash_algo key)
+uint16_t keychain_get_hash_len(enum keychain_hash_algo key)
 {
 	return algo_info[key].length;
```

Comment:
length is still 4 byte value but returned to 2 byte value  ?

Reply:
done